cico_build_deploy.sh

```diff
@@ -39,16 +39,16 @@ echo 'CICO: Deploying to github pages'
 git fetch origin gh-pages
 git checkout -b gh-pages origin/gh-pages
 
-# Move over generated content to new repo 
+# Move over generated content to new repo
 cp -rfv _site/* .
 
 # Update new repo with generated code
 git config user.name "ci.centos.org"
-git config user.email "almighty-public@redhat.com"
+git config user.email "fabric8-services-public@redhat.com"
```

Comment:
I use `fabric8@googlegroups.com` now https://github.com/fabric8-services/fabric8-devdoc/pull/129/files#diff-5bf05017b1ea493887ae831ae52f78ebL47

The email came from a hardcoded sed command, that is why I asked for a revision for the emails, and other details.